docs/editor/settings-sync.md

```diff
@@ -156,6 +156,8 @@ If the keychain throws the error "No such interface "org.freedesktop.Secret.Coll
 
 If the error is "Writing login information to the keychain failed with error 'Unknown or unsupported transport “disabled” for address “disabled:”'", check that `dbus-launch` has been started by following the instructions in [issue #120392](https://github.com/microsoft/vscode/issues/120392#issuecomment-814210643).
 
+If the error is "The name org.freedesktop.secrets was not provided by any .service files", make sure that you have the package `gnome-keyring` installed. Desktop environments other than GNOME do not come with `gnome-keyring` installed, but VS Code depends on it for managing credentials.
```

Comment:
This doesn't depend on `gnome-keyring` but instead it depends there being a package that implements the [Secret Storage API](https://www.gnu.org/software/emacs/manual/html_node/auth/Secret-Service-API.html).

So instead maybe we say:

```suggestion
If the error is "The name org.freedesktop.secrets was not provided by any .service files", make sure that you have a package that implements the [Secret Storage API](https://www.gnu.org/software/emacs/manual/html_node/auth/Secret-Service-API.html) installed, such as `gnome-keyring`. VS Code expects such a package for storing credentials on the machine. More information can be found in [issue #104319](https://github.com/microsoft/vscode/issues/104319#issuecomment-1057588052).
```

Reply:
@TylerLeonhardt Strange... Pretty sure I already had KDE Wallet installed on my system, but Code only stopped giving that error when I installed `gnome-keyring`. But after uninstalling the latter, it seems to still work for me. Even though my GitHub credentials aren't showing up in KWalletManager... :thinking:

Reply:
Yeah I think KDE Wallet hasn't fully implemented the Secret Service API yet. If you take a look at the issue I linked, it was only merged in like a couple weeks ago so likely not showing up in the KDE that most folks have.

Reply:
> But after uninstalling the latter, it seems to still work for me.

Maybe the process is still running and hanging around after install?

Reply:
> Maybe the process is still running and hanging around after install?
ah, I think that was it. When I uninstalled `gnome-keyring`, it still kept around these two packages: `gnome-keyring-pkcs11 libpam-gnome-keyring`
and it seems like one (or both) of those two were still operating & letting Code access the keyring (even after restarting). 

After removing them, Code could still access the keyring until I restarted next.